locale/en/knowledge/intermediate/how-to-log.md

```diff
@@ -57,17 +57,19 @@ Here is an example of setting up a `winston` logger.  This example includes most
 
       require('winston-riak').Riak;
       require('winston-mongo').Mongo;
+      require('winston-couchdb').Couchdb;
```

Comment:
@lpinca No, I meant do we need to list every available transport module in a logging example? I think it's enough to show how to add multiple transports and mention the widely used ones at the end.